### DISCUSSION TITLE
关于书签问题

### DISCUSSION THREAD

**Discussion by Sh1p-wy** [note286/xduts#38] [Q&A] [answered]
首先，感谢您提供的源码，
其次，我有个关于书签的问题想咨询。
![image](https://user-images.githubusercontent.com/30686904/217570507-a932b616-f750-4d74-8fb2-5635b5bd654d.png)
书签只能显示chapter的标题，无法显示子章节的标题，包括section、subsection等下级标题都无法显示。
在xduts.pdf中没能查到相关信息，此外，我确认没有使用\section*{}。

**Comment by Sh1p-wy** [comment]
update，
添加 style/remove-page = {目录}, 后可以正常显示了

**Comment by note286** [comment, marked as answer]
已在`v4.4.3.0`中修复，请从GitHub中下载最新版，替换`xdupgthesis.cls`即可。

**Reply by Sh1p-wy** [reply]
收到， 谢谢。

顺便还有个关于 `图表配置里的表格内容字号` (对应5.12小节) 的问题想咨询下，
在设置 style/table-small-font = true 后，文档仍然是以小四字体显示的，请问这是什么原因

**Reply by note286** [reply]
文档就是小四啊，这是学位办的要求。这个接口修改的是表格内容字号，默认就是`true`，也就是五号。

**Reply by Sh1p-wy** [reply]
我的描述有误，应该是在设置 style/table-small-font = true 后表格内的字体仍然是以小四显示的

**Reply by note286** [reply]
最新版无法复现你的问题。

**Reply by Sh1p-wy** [reply]
get it，我自己再仔细检查一下

**Reply by note286** [reply]
```latex
\documentclass{xdupgthesis}
\xdusetup{style/table-small-font=true}
\begin{document}
\chapter{title}
\begin{table}
\begin{tabular}{cc}
a&a\\
\end{tabular}
\end{table}
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/217710886-3f0852c1-8dfd-4f79-ae1c-10248d1e0255.png)

**Reply by note286** [reply]
`style/table-small-font`的默认值就是`true`，不需要你手动设置。

**Reply by note286** [reply]
如果对这个问题还有问题，请单独开一个讨论。

**Reply by Sh1p-wy** [reply]
找到原因了，是由于使用双栏居中 \begin{table*} 造成的。
再次感谢！

**Reply by note286** [reply]
学位论文只有单栏，无需用`table*`环境啊，这是跨栏的意思，和居中无关，居中请使用`\centering`。

**Reply by Sh1p-wy** [reply]
是的，我疏忽了这个问题。
